site/content/extend/plugins/server/_index.md

```diff
@@ -5,4 +5,36 @@ subsection: Plugins
 weight: 10
 ---
 
-Plugins are capable of extending the Mattermost server by running processes on the server that interact with Mattermost via RPC.
+# Server Plugins
+
+Server plugins are capable of extending the Mattermost server by running processes on the server that interact with Mattermost via remote procedure calls (RPC).
+
+Looking for a quick start? [See our "Hello, world!" guide](/extend/plugins/server/hello-world/).
+
+Want the server plugin reference doc? [Find it here](/extend/plugins/server/reference/).
+
+## Features
+
+#### API
+
+Using the [RPC API](/extend/plugins/server/reference/#api) your server plugin has fast and efficient access to create, read, update and delete operations on server data models.
```

Comment:
I would be a fan of capitalizing "API" and "Hooks" in the URL fragments to be more consistent with the others `/reference/#Hooks.ServeHTTP`.

Reply:
Good call, will do